### PR TITLE
[# 112648433] Create rake task to update epics, and stories that are in flight.

### PR DESCRIPTION
**Pull request by kquet**
[Tracker #112648433](https://www.pivotaltracker.com/story/show/112648433)

### CODE REVIEW

**Comment by kquet** [comment]
@kenberland

File: config/deploy.rb [hunk 1]
```diff
@@ -51,6 +51,14 @@
     task :seed_users, roles: :db, except: {no_release: true} do
       run "cd #{current_path} && bundle exec rake tracker:db:seed_users"
     end
+
+    task :seed_epics, roles: :db, except: {no_release: true} do
+      run "cd #{current_path} && bundle exec rake tracker:db:seed_epics"
```
Comment (skylarcook) [inline config/deploy.rb]: what does `seed_epics` do?

File: lib/tracker/tasks/db.rake [hunk 2]
```diff
@@ -11,5 +11,66 @@ namespace :tracker do
...
+      to_create_story_ids = tracker_active_story_ids - honesty_active_story_ids
+      to_create_story_ids.each do |story_id|
+        puts "creating story_id=#{story_id}"
+        Tracker::Story.create(story_id)
+      end
+    end
+
+    desc 'Update from Tracker'
+    task :update do
+      Rake::Task['tracker:db:update_epics'].execute
```
Comment (soonahn) [inline lib/tracker/tasks/db.rake]: You're calling the Capistrano tasks as Rake tasks.

I think you're confusing the two.

Comment (kquet) [inline lib/tracker/tasks/db.rake]: Comment addressed. @soonahn

**Pull request merged**